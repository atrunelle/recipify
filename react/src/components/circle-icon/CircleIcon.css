.circle-icon {
  margin: auto;
  border-radius: 50%;
  background-color: #fff;
  width: 150px;
  height: 150px;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-bottom: 16px;
  box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12); }
  .circle-icon__icon {
    width: 50px;
    margin-top: 20px; }
