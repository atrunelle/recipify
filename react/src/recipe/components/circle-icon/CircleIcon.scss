.circle-icon {
  margin: auto;
  border-radius: 50%;
  background-color: #fff;
  width: 150px;
  height: 150px;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-bottom: 16px;
  
  &__icon {
    width: 50px;
    margin-top: 20px;
  }
}
