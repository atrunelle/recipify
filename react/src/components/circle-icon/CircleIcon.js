import React, { Component } from 'react';
import './CircleIcon.css';

class CircleIcon extends Component {
  render() {
    return (
      <div className="circle-icon">
        <img 
          className="circle-icon__icon"
          src={this.props.icon}
          alt={this.props.alt}/>
      </div>
    );
  }
}

export default CircleIcon;