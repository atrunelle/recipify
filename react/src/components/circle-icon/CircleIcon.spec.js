import React from 'react';
import { shallow } from 'enzyme';
import CircleIcon from './CircleIcon';

describe('Component: CircleIcon', () => {
  it('should match snapshot', () => {
    const component = shallow(
      <CircleIcon
        icon="'icon'"
        alt="'alt text"
      />
    );
    expect(component.html()).toMatchSnapshot();
  });
});