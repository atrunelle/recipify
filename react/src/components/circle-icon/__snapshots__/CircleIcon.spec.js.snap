// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: CircleIcon should match snapshot 1`] = `"<div class=\\"circle-icon\\"><img class=\\"circle-icon__icon\\" src=\\"&#x27;icon&#x27;\\" alt=\\"&#x27;alt text\\"/></div>"`;
